src/CssVariables/index.js

```diff
@@ -1,16 +1,19 @@
 import React from 'react'
 import propTypes from 'prop-types'
-import * as theme from '../theme.js'
 
 /**
  * @module
- * @description Injects our theme as CSS custom properties
+ * @description Injects the variables prop as CSS custom properties
  * @param {CssVariables.PropTypes} props
  * @returns {React.Component}
  * @example import { CssVariables } from @dhis2/ui-core
  * @see Live demo: {@link /demo/?path=/story/cssvariables--default|Storybook}
  */
 const CssVariables = ({ variables }) => {
+    if (!variables) {
```

Comment:
I think this is unnecessary when marking the prop as required.

Reply:
I wanted to return early here because the mapping will crash if there is no variable prop.

Reply:
If the variables are required, crashing would be a "desired" behavior, wouldn't it?

Reply:
I do find it a bit extreme. This is **without** handling a lack of variables:

<img width="1061" alt="Screenshot 2019-09-16 at 14 59 59" src="https://user-images.githubusercontent.com/7355199/64960157-03612100-d893-11e9-84a6-8a83fbb812fb.png">

What will be displayed depends on the error handling the app does of course. And alongside this error there are also various other error messages in the console (that are a bit more helpful).

And this is **with** handling a lack of variables (so just a console message):

<img width="1278" alt="Screenshot 2019-09-16 at 15 00 35" src="https://user-images.githubusercontent.com/7355199/64960326-5e931380-d893-11e9-8e94-47717f233801.png">

I guess it's not incorrect to crash when a component's API isn't correctly used. I do always try to neatly handle any potential errors though, and just the prop-type validation failure does seem a bit more civil to me. But I don't have a strong opinion on this, I can see the point of both approaches.

Reply:
On second thought, just letting it throw doesn't seem too bad either. I'll leave it to what @varl prefers, I'm fine with either direction.

Reply:
This entire problem can be sidestepped by having a single component that does not accept any props.